rules/xcodeproj.bzl

```diff
@@ -236,7 +260,7 @@ def _xcodeproj_aspect_impl(target, ctx):
 
 _xcodeproj_aspect = aspect(
     implementation = _xcodeproj_aspect_impl,
-    attr_aspects = ["deps", "actual", "tests", "infoplists", "entitlements", "resources", "test_host"],
+    attr_aspects = ["deps", "actual", "tests", "infoplists", "entitlements", "resources", "test_host", "families"],
```

Comment:
given that `families` isn't a `label{,list}` attribute, i dont think we need to propagate through it?